create_infotexts.py

```diff
@@ -112,83 +112,16 @@ def create_smvk_mm_link(item):
-    :return:  string representing the selcted mapping value
-    """
-    if "wikidata" in people_mapping[flipped_name].keys():
-        return people_mapping[flipped_name]["wikidata"]
-    elif "commons" in people_mapping[flipped_name].keys():
-        return people_mapping[flipped_name]["commons"]
-    else:
-        return people_mapping[flipped_name]["name"]
-
-
-def extract_mappings_from_list_of_depicted_people(flipped_names):
-    """
-    Pass each name in a list of names to mapping function.
-
-    :param flipped_names: string with full names turned around from e.g. "surname, given name" -> "given name surename"
-    :return: a string represention of several names mapped to either wikidata, commons or the names only
-    """
-
-    out_string = ""
-    for name in flipped_names:
-        selected_mapping = select_best_mapping_for_depicted_person(name)
-        out_string += selected_mapping + "/"
-    mapped_names = out_string.rstrip("/")
-    return mapped_names
-
-
-def extract_mapping_of_depicted_person(flipped_name):
-    """
-        :flipped_name: string representing full name turned around from e.g.
-        "surname, given name" -> "given name surename"
-        :return: string representation of a name mapped to either wikidata, commons or the name only
-        """
-    mapped_name = select_best_mapping_for_depicted_person(flipped_name)
-    return mapped_name
-
-
-def map_depicted_person_field(name_string_or_list):
-    """
-        :name_string_or_list: string representing one full name or series of full names (of faulty)
-        :return: a string of either one mapped name or several mapped names
-        """
-    words = name_string_or_list.split(", ")
-    span = 2
-    joined_words = [", ".join(words[i:i + span]) for i in range(0, len(words), span)]
-    # print("joined_words: {}".format(joined_words))
-
-    if len(joined_words) == 1:
-        flipped_name = helpers.flip_name(joined_words[0])
-        # print("flipped_name: {}".format(flipped_name))
-        depicted_people_value = extract_mapping_of_depicted_person(flipped_name)
-        return depicted_people_value
-
-    elif len(joined_words) > 1:
-        flipped_names = helpers.flip_names(joined_words)
-        # print("flipped_names: {}".format(flipped_names))
-        depicted_people_value = extract_mappings_from_list_of_depicted_people(flipped_names)
-        return depicted_people_value
-
-    else:
-        # TODO: add logic for maintanence category for faulty depicted people field
-        print("<Personnamn / avbildad> doesn't seem to be even full names: {}".format(name_string_or_list))
-        return name_string_or_list
-
-
-def generate_infobox_template(item, places):
+def generate_infobox_template(item, img):
     """Takes one item from metadata dictionary and constructs the infobox template.
-    :item: one metadata row for one photo
+    :param item: one metadata row for one photo
+    :param places: 
```

Comment:
Remove this line if the `places` argument will not be used (see comment below though).